### ISSUE TITLE
Install fails with exact version of a dependency

### ISSUE THREAD

**Issue by aecz** [watilde/dep#11]
### Steps to reproduce behavior
```
npm i -g @angular/cli@1.3.2
ng new test-dep-v0.14.1 --skip-install
cd test-dep-v0.14.1
dep i
```
or using npx to not install @angular/cli
```
npx @angular/cli@1.3.2 new test-dep-v0.14.1 --skip-install
cd test-dep-v0.14.1
dep i
```

### Expected behavior
Install dependencies into node_modules.

### Actual behavior
```
$ dep i
Resolving dependencies
DEBUG: { url: 'https://registry.yarnpkg.com/@angular/cli/1.3.2',
  headers: { 'User-Agent': 'dep/0.14.1 node/v6.11.2' } }
SyntaxError: Unexpected end of JSON input
    at Object.parse (native)
    at Request.request.get.on.on.e (/Users/aecz/.nvm/versions/node/v6.11.2/lib/node_modules/dep/lib/install/resolver/fetchers/registry.js:20:23)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Users/aecz/.nvm/versions/node/v6.11.2/lib/node_modules/dep/node_modules/request/request.js:1091:12)
    at IncomingMessage.g (events.js:292:16)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
```
I added the DEBUG log to show the `options`variable.
I works changing the URL to:
```
https://registry.yarnpkg.com/@angular%2fcli/=1.3.2
```
I also faced the issue #10 as you can see.

**Comment by watilde** [comment]
Hello!

Your issue report is perfect. I could reproduce it easily, thanks :) Even I've fixed #10, this bug is still able to be reproduced so I will work on this.

**Comment by watilde** [comment]
The patch was released as ` v0.14.2` and this issue should be fixed. Thanks again for reporting the issue with the full information<3 Cheer!
